shell_integration/dolphin/ownclouddolphinpluginhelper.cpp

```diff
@@ -62,18 +62,17 @@ void OwncloudDolphinPluginHelper::sendCommand(const char* data)
 
 void OwncloudDolphinPluginHelper::sendGetClientIconCommand(int size)
 {
-    QByteArray line_end("\n");
-    QJsonObject args { { "size", size } };
-    QJsonObject obj { { QStringLiteral("id"), "1" }, { QStringLiteral("arguments"), args } };
-    std::string command = "V2/GET_CLIENT_ICON:" + QJsonDocument(obj).toJson(QJsonDocument::Compact).toStdString();
-    sendCommand(QByteArray(command.c_str() + line_end));
+    int msgId = 1;
```

Comment:
Here I meant a member that counts upwards.
the v2 response will have the same id, so when you (in theory) start multiple requests you can map them

Reply:
yes, I see